src/OrbitQt/ConnectToStadiaWidget.ui

```diff
@@ -205,14 +235,17 @@
                 <property name="visible">
                  <bool>false</bool>
                 </property>
-                <property name="accessibleName">
-                 <string>Overlay</string>
-                </property>
+				<property name="accessibleName">
```

Comment:
I assume most of these changes are done by Designer automatically?

This specific whitespace thing here seems to be done by hand? Maybe another save in Designer will format this correctly

Reply:
Thanks for catching that, I think I added that by hand indeed. Saving the file fixed it.